core/src/scheme/mod.rs

```diff
@@ -15,8 +15,6 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 mod genesis;
-mod pod_account;
```

Comment:
If you remove the `pod_account` then you can remove the serialized initial accounts information from `.json` scheme files.